src/transaction/locator_sr.c

```diff
@@ -13359,7 +13359,7 @@ locator_get_object (THREAD_ENTRY * thread_p, const OID * oid, OID * class_oid, R
 	{
 	  ASSERT_ERROR ();
 	  heap_clean_get_context (thread_p, &context);
-	  return ER_FAILED;
+	  return S_DOESNT_EXIST;
```

Comment:
@andrei14vl @ribram 

Is it all right to return `S_DOESNT_EXIST` for all cases?
Please imagine `heap_prepare_object_page` fails to fix the page due to interrupt.